.github/workflows/lint-test-build-push.yaml

```diff
@@ -150,8 +143,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          pip install coveralls~=2.1
-          coveralls
+          sudo pip install coveralls~=2.1
+          sudo coveralls
```

Comment:
Is there any reason why we don't make a user install rather than a system install?

Reply:
IIRC it's cause the user site is not on PATH. It's more convenient to use `sudo` in this case.

Reply:
Though maybe it's bad practice, even in CI, and I should have kept using the Python version task...

Reply:
Don't yell at me, but I forced push to get rid of it. Turns out I had already dropped the commit locally last month, forgot about it, and commited changes over it.

Reply:
Cool!